overlord/snapstate/snapstate_test.go

```diff
@@ -9044,47 +9044,115 @@ type switchScenario struct {
 	summary  string
 }
 
-var switchScenarios = []switchScenario{
```

Comment:
one thing that might be confusing: before this change, an empty cohTo was shorthand for "no change", whereas now it means "leave cohort".